pkg/certificate/manager_test.go

```diff
@@ -68,14 +54,55 @@ var _ = Describe("Test Certificate Manager", func() {
 	})
 })
 
+var _ = Describe("Test Rotor", func() {
+
+	cn := CommonName("foo")
+
+	Context("Testing rotating expiring certificates", func() {
+
+		validityPeriod := -1 * time.Hour // negative time means this cert has already expired -- will be rotated asap
+
+		stop := make(chan struct{})
+		defer close(stop)
+		msgBroker := messaging.NewBroker(stop)
+		certManager, err := NewManager(&Certificate{}, &fakeIssuer{}, validityPeriod, msgBroker)
+		certManager.Start(5*time.Second, stop)
```

Comment:
If we're not running `checkAndRotate` at the beginning of the loop this kicks off, then this 5s is basically a lower bound on how fast the test can run. I'd either do a `checkAndRotate` immediately in the loop or make this timeout (only in this test here) something way smaller, like 10ms.

Reply:
Oh I thought I fixed this but I stashed it - adding this outside the for loop 🤦‍♀️